relay-server/src/actors/project_buffer.rs

```diff
@@ -302,12 +318,16 @@ impl BufferService {
                     relay_log::error!("failed to serialize the envelope: {}", LogError(&err));
                 }
             });
-            query_builder.build().execute(db).await?;
+            let result = query_builder.build().execute(db).await?;
+            relay_statsd::metric!(counter(RelayCounters::BufferWrites) += 1);
```

Comment:
This will generate a ton of metrics. Can we use a counter here for each loop iteration, and log the metric outside of the loop? The result should be pretty similar, with less overhead.

Reply:
I considered it, but I wanted the metric to be as real time as possible. This logs one metric for each batch of 333 envelopes, so I hope it's ok.

Reply:
Seems reasonable, yes 👍